tests/phpunit/data/html5-entities/README.md

```diff
@@ -0,0 +1,22 @@
+# HTML5 Entities
+
+This directory contains the listing of HTML5 named character references and a script that can be used
+to create or update the optimized form for use in the HTML API.
+
+The HTML5 specification asserts:
+
+> This list is static and will not be expanded or changed in the future.
+>  - https://html.spec.whatwg.org/#named-character-references
+
+The authoritative [`entities.json`](https://html.spec.whatwg.org/entities.json) file comes from the WHATWG server, and
+is cached here in the test directory so that it doesn't need to be constantly re-downloaded.
+
+## Updating the optimized lookup class.
+
+The `html5-named-character-references.php` file contains an optimized lookup map for the entities in `entities.json`.
+Run the `generate-html5-named-character-references.php` file to update the auto-generated Core module.
+
+```bash
+~$ php tests/phpunit/data/html5-entities/generate-html5-named-character-references.php
+OK: Successfully generated optimized lookup class.
+```
```

Comment:
This is super helpful.

Maybe we could make it a bit clearer exactly what file this is pointing to and even link it (I _think_ that's correct syntax):

````suggestion
The [`html5-named-character-references.php`][1] file contains an optimized lookup map for the entities in `entities.json`.
Run the `generate-html5-named-character-references.php` file to update the auto-generated Core module.

```bash
~$ php tests/phpunit/data/html5-entities/generate-html5-named-character-references.php
OK: Successfully generated optimized lookup class.
```

- [1]: ../../../../src/wp-includes/html-api/html5-named-character-references.php
````